data/mods/gen9ssb/conditions.ts

```diff
@@ -663,6 +672,111 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 		noCopy: true,
 		// phrases TBD
 	},
+	partman: {
+		noCopy: true,
+		onStart(pokemon) {
+			let message;
+			switch (this.toID(enemyStaff(pokemon))) {
+			case 'partman':
+				message = 'Hii Q - oh, it\'s just me.';
+				break;
+			case 'hydrostatics':
+				message = 'Here to bully Hydro';
+				break;
+			case 'softflex':
+				message = '/me softly flexes';
+				break;
+			case 'computerwizard8800':
+				message = 'CWIZ SLEEP';
+				break;
+			case 'krytocon':
+				this.add(`c:|${getName('PartMan')}|%r 14 // @Krytocon`);
+				this.add(`c:|${getName('Ice Kyubs')}|Roll: 14`);
+				message = null;
+				break;
+			case 'kennedy':
+				message = 'Down the reds!';
+				break;
+			case 'mex':
+				message = 'Probopass moment';
+				break;
+			case 'monkey':
+				message = 'Remember to smile!';
+				break;
+			case 'pissog':
+				message = 'Ma ciaomi queste noci';
+				break;
+			case 'sulo':
+				message = '...Sulo\'s AFK again, aren\'t they?';
+				break;
+			case 'warriorgallade':
+				message = 'Berry nice to meet you!';
+				break;
+			case 'breadloeuf':
```

Comment:
they namedchanged

Reply:
To be fair Breadbowl had a badly namechange

Reply:
Resolved in https://github.com/smogon/ps-ssb/pull/86/commits/f583c2e6bcb8b691ab36a9991cd7ebd5608f5126